submodules/wizard/views/step-accountContacts.html

```diff
@@ -0,0 +1,58 @@
+{{#*inline "inputPartial"}}
+	<div class="span4">
+		<label for="{{name}}">{{coalesce i18n.label name}}</label>
+		<input class="input-block-level{{#compare mask "===" "phoneNumber"}} phone-number{{/compare}}" type="{{#if type}}{{type}}{{else}}text{{/if}}" name="{{name}}" id="{{name}}"{{#if (coalesce i18n.placeholder i18n.label)}} placeholder="{{coalesce i18n.placeholder i18n.label}}"{{/if}}{{#if required}} required{{/if}}{{#if mask}} data-mask="{{mask}}"{{/if}} value="{{lookupPath @root.data name}}">
+	</div>
+{{/inline}}
+
+<div id="step_account_contacts" class="wizard-step container-fluid">
+	<div class="step-title">
+		{{i18n.accountsApp.wizard.steps.accountContacts.title}}
+	</div>
+	<div class="step-body">
+		<form id="form_account_contacts" name="form_account_contacts">
+			<div class="step-section">
+			{{#with i18n.accountsApp.wizard.steps.accountContacts as |accountContactsI18n|}}
+				{{#with @root.i18n.accountsApp.wizard.steps.general as |generalI18n|}}
+				<div class="section-title">
+					{{accountContactsI18n.salesRep.title}}
+				</div>
+				<div class="row-fluid">
+					<div class="span4">
```

Comment:
Flexbox + percentage width + media queries could have been used for a more custom layout here, think about it next time.

Reply:
I thought about it, but then remembered that Bootstrap was already available, and opted for it to not to increase the CSS styles. I will use custom styles for the next steps, and retrofit this and the previous one.